sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobUrlParts.java

```diff
@@ -17,17 +18,13 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.TreeMap;
-import java.util.regex.Pattern;
 
 /**
  * This class represents the components that make up an Azure Storage Container/Blob URL. You may parse an
  * existing URL into its parts with the {@link #parse(URL)} class. You may construct a URL from parts by calling {@link
  * #toUrl()}.
  */
 public final class BlobUrlParts {
-    private static final Pattern IP_V4_URL_PATTERN = Pattern
-        .compile("(?:\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})|(?:localhost)");
-
     private final ClientLogger logger = new ClientLogger(BlobUrlParts.class);
 
     private String scheme;
```

Comment:
I think the docs for setHost() should be updated to give an example of ip style

Reply:
Added an example of the IP style URL Azurite expects.